Italian/t-rex/cadi.md

```diff
@@ -0,0 +1,5 @@
+Ti stai calando dalla finestra, quando la fune si spezza e precipiti nel vuoto
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.